### PR TITLE
Update change log 

### PR DESCRIPTION
**Pull request by backwind1233**
change log entry can not be empty, or it can't process spring release pipeline.

### CODE REVIEW

File: sdk/spring/azure-identity-spring/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.6.0 (2021-06-23)
-
+### New Features
```
Comment (chenrujun) [inline sdk/spring/azure-identity-spring/CHANGELOG.md:4]: This is not a feature, how about change it to `### Dependency Upgrades`

Refs: https://github.com/spring-projects/spring-boot/releases

Comment (chenrujun) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:4]: And this PR has 18 files changed, I think we will release more than 18 artifacts. Did you need to change more files?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-identity-spring/CHANGELOG.md (hunk 1 above)
Comment (backwind1233) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:4]: I checked it one by one, I changed only files with empty entry

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**